Pokedex_ios/View/Controller/PokemonList/PokemonListViewController.swift

```diff
@@ -26,26 +26,39 @@ class PokemonListViewController: UIViewController {
         setBackground()
         initViewModel()
         setSearchView()
-        // Setup for the advanced filter image and button
-        setUpAdvanceFilterPopup()
+        // Setup for all filter image and button
+        setFiltersPopup()
```

Comment:
better use your own method, leave alone setupAdvanceFilterPopup and make one that is called setupGenerationPopup. Mostly for better code reading purposes

Reply:
i can't add the icons to the navigationItem separately but i changed the name of the method to be more specific